Source/Help/Output/articles/Tutorials.html

```diff
@@ -5,11 +5,11 @@
   <head>
     <meta charset="utf-8">
       <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-      <title> | Krypton Component Suite 2020
+      <title>Standard Toolkit Tutorials | Krypton Component Suite 2020
 BSD 3-Clause License
 � Component Factory Pty Ltd, 2006 - 2024, All rights reserved. </title>
```

Comment:
This is the wrong end year for the component factory !

Reply:
Now fixed :)